src/components/common.js

```diff
@@ -523,18 +523,27 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) =
         backgroundRepeat: 'no-repeat', backgroundSize: '750px', backgroundPosition: 'right 0 top 0'
       }
     }, [
-      h1({ style: { fontSize: 54 } }, `Welcome to ${getAppName()}`),
+      h1({ style: { fontSize: 54 } }, [
+        'Welcome to ',
+        span({ style: { display: isTerra() ? 'block' : 'inline-block' } }, getAppName(isTerra()))
+      ]),
       div({ style: { margin: '1rem 0', width: 575, ...(bigSubhead ? { fontSize: 20, lineHeight: '28px' } : { fontSize: 16, lineHeight: 1.5 }) } }, [
-        `${getAppName(true)} is a ${Utils.cond(
+        `${getAppName(!isTerra())} is a ${Utils.cond(
```

Comment:
Not totally sure where this gets used but is it expected that it's `!isTerra()` here but `isTerra()` on line 528?

Reply:
It's confusing :/ but expected. Basically there's a longName and a shortName for the site. For Terra we want:
```
Welcome to Terra Community Workbench! [long name]

Terra [short name] is a…
```

For AnVIL (and other sites) it's reverse:
```
Welcome to AnVIL! [short name]

The NHGRI AnVIL (Genomic Data Science Analysis, Visualization, and Informatics Lab-space) [long name] is a …
```